README.md

```diff
@@ -26,21 +26,74 @@ following rules, which we consider useful.
 
 ## Belt Activities
 
-The belt activities serve as a checklist for the team and give additional information to accomplish these activitites. In particular, they consist of:
-- a brief task description the team,
+The belt activities serve as a checklist for the team and give additional information to accomplish these activities. In particular, they consist of:
+- a brief task description for the team,
 - a list of further readings to learn more about this topic,
-- references to related activities (if necessarry),
+- references to related activities (if necessary),
 - the benefits of this activity, and
-- a checklist for the asssessment of the belt.
+- a checklist for the assessment of the belt.
 
-## More Headings
+## Roles
 
-TODO: Assessment and other topics
+Security Belts depends on the existence of several roles from the Agile methodology such as Developer and Product Owner (PO). Developers, PO, and optionally a scrum master, form the so-called *team*. 
+
+### Security Champion
+
+The maturity model Security Belts adds the role of the Security Champion to the Agile methodology. The Security Champion is a developer, who is also part of the team. He/she has a profound security knowledge but is not necessarily a security expert. The Security Champion empowers the team such that they can ensure the security of their software products by themselves. Moreover, he/she enables the team to improve themselves concerning software security and raises the importance of security within the team (incl. PO) as well as towards the management.
+
+Security Champions must attend a thorough training. They need an overview of the complete security development lifecycle (SDLC), theoretical background, 
+practical knowledge, and the necessary soft skills to fulfill its role. Examplary trainings are: 
+<!--
+- [Fraunhofer IEM - Security Champion Training](https://www.iem.fraunhofer.de/de/academy/schulungsangebot/security-champion-training.html)
+- TODO
+-->
+
+### Security Champion Guild
+
+The Security Champion Guild is a guild as known from the [Spotify model](https://www.atlassian.com/agile/agile-at-scale/spotify). Security Champions participate to get feedback on security-related topics from outside their teams such as individual questions, second opinions, and code/concept reviews.
+
+The main task of the guild is to spread the Security Belts in the organization. The guild conducts belt assessments and actively improves the maturity model.
+
+All Security Champions and only these are part of the guild. Otherwise, the champions would not learn from each other and the quality of the guild's outcome is at risk as Security Champions have a dedicated training, which other developers have not. If a central security department exists within the company, then the guild and this department should collaborate.
```

Comment:
```suggestion
All Security Champions and only these are part of the guild. Otherwise, the champions would not learn from each other and the quality of the guild's outcome is at risk as Security Champions have a dedicated training, which other developers have not. If a central security department exists within the company, then the guild and this department need to collaborate.
```